dashboard/lib/services/curriculum_pdfs.rb

```diff
@@ -1,13 +1,32 @@
 require 'cdo/chat_client'
 require 'dynamic_config/dcdo'
 
+# Overview of all Curriculum PDFs
+#
+# Current:
+# - teacher lesson plans (csp1-2021/20210216001309/teacher-lesson-plans/Welcome to CSP.pdf)
+# - student lesson plans (csp1-2021/20210216001309/student-lesson-plans/Welcome to CSP.pdf)
+# - script resources (csp1-2021/20210216001309/resources/CSP Unit 1 - Digital Information ('21-'22) - Resources.pdf)
+# - script overview (csp1-2021/20210216001309/CSP Unit 1 - Digital Information ('21-'22).pdf)
```

Comment:
Do we also want to include the course overview in the all lessons PDF when you get it from the course page?

Reply:
I guess maybe said a different way it looks like this adds the script overview pdf. Do we need a unit group overview (course overview) pdf which is a combo of the script overview pdfs and the course overview page pdf?

Reply:
@tess323 Any thoughts on this?

Reply:
@dmcavoy - Looking at CB we do not currently have this functionality so it doesn't feel like P0 to me. That said I think it makes sense in theory to have something like this if we could get it for cheap.  My priorities are:

- getting PDFs that roughly match curriculum builder live so we can run them by PL 
- adding any additional PDFs we think we can sneak in under the wire